andela/activo-api
# #159740438 Update requirements.txt to resolve security vulnerabilities

#### What does this PR do?
Updates cryptography version in the requirements.txt file to version 2.3.

#### Description of Task to be completed?
Update the cryptography version in requirements.txt file to version 2.3 to resolve security issues in earlier versions.

#### How should this be manually tested?
1. Do a `git fetch origin ch-update-requirements-file-159740438` to fetch the remote branch
2. Do a `git checkout ch-update-requirements-file-159740438` to checkout to your branch
3. Check the version of cryptography in the requirements.txt file. It should read as cryptography==2.3.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159557184](https://www.pivotaltracker.com/story/show/159740438)

#### What pivotal tracker stories does this block?
None

#### What pivotal tracker stories are blocking this?
None

### Postman Documentation
N/A

#### Screenshots (if appropriate)
N/A
